app/controllers/claims/schools/mentors_controller.rb

```diff
@@ -1,7 +1,26 @@
 class Claims::Schools::MentorsController < ApplicationController
   include Claims::BelongsToSchool
+  before_action :set_mentor, only: %i[show remove destroy]
 
   def index
     @pagy, @mentors = pagy(@school.mentors.order(:first_name, :last_name))
   end
+
+  def show; end
+
+  def remove; end
+
+  def destroy
+    mentor_membership = @mentor.mentor_memberships.find_by(school: @school)
+    mentor_membership.destroy!
```

Comment:
If a membership isn't found, the following line will fail with a "`nil` does not respond to `destroy!`".

Let's escape that early.

```suggestion
    mentor_membership = @mentor.mentor_memberships.find_by!(school: @school)
    mentor_membership.destroy!
```

We'll now render a 404 page if the membership doesn't exist.